src/containers/CategoryFilter.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cat } from './BookForm';
```

Comment:
presentational components should be placed inside `src/components` folder [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/presentationalCategoryFilter.png)]

This was mentioned in milestone 2, we should've corrected there

Please relocate the file